Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:16.04
 
-MAINTANER Shagun Khemka "shagun.khemka60@gmail.com"
+#MAINTANER Shagun Khemka "shagun.khemka60@gmail.com"
```

Comment:
@pushkalkatara Please revert this change

Reply:
@S2606 Could you please explain why we need this line?

Reply:
Sorry my bad!

Reply:
Docker build was giving an error with the line. No idea why.

Reply:
@pushkalkatara @S2606 Can we remove this line?

Reply:
@pushkalkatara It was because maitainer was spelled incorrectly.

Reply:
Oh okay. I'm working on the above conflicts. I am new to open source, suggestions in code style etc are appreciated.